### PR TITLE
DISTX-739 Fix EnableLoadBalancer set to false irrespective of true in custom cluster definition

### PR DESCRIPTION
**Pull request by Vatsal23082000**
See detailed description in the commit message.

### CODE REVIEW

**Review by sidseth — COMMENTED** [review]
@Vatsal23082000  - I don't know this code well enough to review this.

That said, I'm a little surprised that a database schema change is required for this - given that the default shapes already work with load balancers.
Was under the impression that this was just a missing field in one of the definitions.

File: core/src/main/resources/schema/app/20230110074614_DISTX-739_Fix_EnableLoadBalancer_set_to_false_irrespective_of_true_in_custom_cluster_definition.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- // DISTX-739 Fix EnableLoadBalancer set to false irrespective of true in custom cluster definition
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE clustertemplate ADD COLUMN IF NOT EXISTS enableloadbalancer BOOLEAN;
```
Comment (doktoric) [inline core/src/main/resources/schema/app/20230110074614_DISTX-739_Fix_EnableLoadBalancer_set_to_false_irrespective_of_true_in_custom_cluster_definition.sql:4]: dont we need explicit default value for this field?

Comment (Vatsal23082000) [reply core/src/main/resources/schema/app/20230110074614_DISTX-739_Fix_EnableLoadBalancer_set_to_false_irrespective_of_true_in_custom_cluster_definition.sql]: I have set the default to true

**Review by doktoric — APPROVED** [review]

**Review by satyabollineni — APPROVED** [review]
LGTM

**Pull request merged**